back/tests/cmd/openbsrv-integration/integration_test.go

```diff
@@ -0,0 +1,445 @@
+package integrationTests
+
+import (
+	"context"
+	"fmt"
+	"github.com/OpenEugene/openboard/back/internal/pb"
+	"google.golang.org/grpc"
+	"testing"
+)
+
+func TestClientServices(t *testing.T) {
+	ctx := context.Background()
+	conn, err := grpc.Dial(":4242", grpc.WithInsecure())
+	if err != nil {
+		t.Log(err)
+		t.Fail()
+	}
+	defer conn.Close()
+
+	// User Service Tests
+	userClnt := pb.NewUserSvcClient(conn)
+	t.Run("Add and find user role", userSvcAddAndFndRoleFunc(ctx, conn, userClnt))
+	t.Run("Add and find users", userSvcAddAndFndUsersFunc(ctx, conn, userClnt))
+	t.Run("Find and delete user", userSvcDelUserFunc(ctx, conn, userClnt))
+
+	// Post Service Tests
+	postClnt := pb.NewPostClient(conn)
+	t.Run("Add type", postSvcAddAndFndTypesFunc(ctx, conn, postClnt))
+	t.Run("Add and find posts", postSvcAddAndFndPostsFunc(ctx, conn, postClnt))
+	t.Run("Add, find and edit post", postSvcEdtPostFunc(ctx, conn, postClnt))
+	t.Run("Find and delete post", postSvcDelPostFunc(ctx, conn, postClnt))
+}
+
+func userSvcAddAndFndRoleFunc(ctx context.Context, conn *grpc.ClientConn, clnt pb.UserSvcClient) func(*testing.T) {
```

Comment:
Please swap the "Func" suffix for "Fn". User-defined types often use the Func suffix, and I suggest using the "Fn" suffix when a variable holds or a function returns a function. If a variable or function is meant to be called directly, I suggest not having a suffix, even if it occasionally gets pass around as a value without being called.

Reply:
Done.